cockroachdb/src/jepsen/cockroach.clj

```diff
@@ -908,7 +908,7 @@
      :f     :transfer
      :value {:from   (rand-int n)
              :to     (rand-int n)
-             :amount (rand-int 5)}}))
+             :amount (1 + rand-int 4)}}))
```

Comment:
I've never written Clojure, but from the minuscule knowledge I have of Racket, I think the plus sign may come in front.